squad/frontend/tests.py

```diff
@@ -122,11 +124,22 @@ def get(cls, build, page, search, per_page=50):
         memo = {}
         for test in tests:
             memo.setdefault(test.full_name, {})
-            memo[test.full_name][test.test_run.environment_id] = test.status
+            memo[test.full_name][test.test_run.environment_id] = [test.status]
```

Comment:
I think the code will be a lot more clear if we use a dict with proper names for the fields instead of an array. at some point decoding the [0], [1], [2] and [3] everywhere can get confusing